### DISCUSSION TITLE
how can i transform pyproj

### DISCUSSION THREAD

**Discussion by chrisks02** [pyproj4/pyproj#1088] [Q&A] [answered]
- grid_mapping_name = "lambert_conformal_conic(WGS84))";
- standard_parallel1 = 30.0; // double
- standard_parallel2 = 60.0; // double
- origin_latitude = 38.0; // double
- central_meridian = 126.0; // double
- false_easting = 0.0; // double
- false_northing = 0.0; // double
- image_width = 900L; // long
- image_height = 900L; // long
- pixel_size = 2000.0; // double
- upper_left_easting = -899000.0; // double
- upper_left_northing = 899000.0; // double
- upper_right_easting = 899000.0; // double
- upper_right_northing = 899000.0; // double
- lower_left_easting = -899000.0; // double
- lower_left_northing = -899000.0; // double
- lower_right_easting = 899000.0; // double
- lower_right_northing = -899000.0; // double

**Comment by snowman2** [comment, marked as answer]
The name you have seems non-standard, otherwise I would suggest: https://pyproj4.github.io/pyproj/stable/build_crs_cf.html

You will likely need to build it manually:
- https://pyproj4.github.io/pyproj/stable/build_crs.html
- https://pyproj4.github.io/pyproj/stable/api/crs/coordinate_operation.html#pyproj.crs.coordinate_operation.LambertConformalConic2SPConversion
